src/WebJobs.Script/Description/Compilation/ICompilationService.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Abstractions.Description;
```

Comment:
Nit: New using statements. Needed?

Reply:
Yes, because `FunctionMetadata` was in `Microsoft.Azure.WebJobs.Script.Description` and moved to `Microsoft.Azure.WebJobs.Script.Abstractions.Description`